kyaml/go.mod

```diff
@@ -5,21 +5,21 @@ go 1.16
 require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/go-errors/errors v1.0.1
-	github.com/go-openapi/spec v0.19.5
-	github.com/go-openapi/strfmt v0.19.5
-	github.com/go-openapi/validate v0.19.8
 	github.com/google/go-cmp v0.4.0
+	github.com/mailru/easyjson v0.7.0 // indirect
 	github.com/markbates/pkger v0.17.1
 	github.com/monochromegane/go-gitignore v0.0.0-20200626010858-205db1a8cc00
 	github.com/pkg/errors v0.9.1
 	github.com/sergi/go-diff v1.1.0
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5 // indirect
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/objx v0.2.0 // indirect
+	github.com/stretchr/testify v1.5.1
 	github.com/xlab/treeprint v0.0.0-20181112141820-a009c3971eca
 	go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5
 	gopkg.in/yaml.v2 v2.4.0
-	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c
+	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776
```

Comment:
This is being automatically updated during `go mod tidy` because of https://github.com/kubernetes-sigs/kustomize/blob/14d2f4bb15cb6513788f2590840a62922c6e66e7/api/go.sum#L264-L265 (the version we also use in k/k: https://github.com/kubernetes/kubernetes/blob/b81a36021f0a6d3d4944c3aba779e0174955515c/go.mod#L492). That's why I added the `replace` directive.